docs/subnets/customize-a-subnet.md

```diff
@@ -80,6 +80,12 @@ The default genesis Subnet-EVM provided below has some well defined parameters:
 }
 ```
 
+For an example of a `genesis.json` file used to test GoEthereum, see 
```

Comment:
Suggest to put it as an :::info per https://docusaurus.io/docs/2.2.0/markdown-features/admonitions